README.md

```diff
@@ -88,12 +88,14 @@ While Golang, Docker and OVS can run on many operating systems, we are currently
 * Clustering over NAT adapter will not work. Hence, the Virtualbox VMs must have either **Host-Only Adapter (or) Internal Network (or) Bridged adapter** installed for clustering to work.
 * The VMs/Hosts must have **unique hostname**. Make sure that /etc/hosts in the VMs have the unique hostname updated.
 
-    curl -sSL http://get.socketplane.io/ | sudo sh
+    curl -sSL http://get.socketplane.io/ | sudo BOOTSTRAP=true sh
 
 or
 
-    wget -qO- http://get.socketplane.io/ | sudo sh
+    wget -qO- http://get.socketplane.io/ | sudo BOOTSTRAP=true sh
 
+>Note:  The first node in the cluseter should set the BOOTSTRAP=true variable.  All other nodes will default to BOOTSTRAP=flase in the install script. 
```

Comment:
Typo `cluseter`. The warning should state that BOOTSTRAP=true should be used on the *first node only*. Without it, it won't work. If used on subsequent nodes, bad things will happen. E.g God killing a 🐱